mkroosevelt.js

```diff
@@ -8,20 +8,26 @@ function showHelp() {
   console.log("");
   console.log("USAGE:");
   console.log("");
-  console.log("create an app in this directory:");
+  console.log("create an app in the current directory:");
+  console.log("mkroosevelt .");
+  console.log("");
+  console.log("create an app in this directory in a new folder:");
   console.log("mkroosevelt appName");
   console.log("");
   console.log("create an app somewhere else:");
   console.log("mkroosevelt /path/to/appName");
 }
 
-function createSampleApp() {
+function createSampleApp(currentDirectory) {
   var fs = require('fs'),
       path = require('path'),
       wrench = require('wrench');
-      
+
+  if (currentDirectory === true) {
+    cmd = path.normalize(process.cwd());
+  }
   wrench.copyDirSyncRecursive(path.normalize(__dirname + '/sampleApp/'), path.normalize(cmd), {
-    forceDelete: false, // Whether to overwrite existing directory or not
+    forceDelete: currentDirectory, // Whether to overwrite existing directory or not. Use currentDirectory as variable because this needs to be set to true if the user wants to create an app in the current directory.
```

Comment:
This is [highly destructive](https://github.com/ryanmcgrath/wrench-js/blob/13f486d867d28fb72947c06f57ce4e5b8e656e86/lib/wrench.js#L236).

We need to warn the user that it will destroy everything in the current directory and ask if that's okay.